src/theme/textStyles.ts

```diff
@@ -0,0 +1,80 @@
+export const textStyles = {
+  "display-1": {
+    fontSize: "4rem",
+    fontWeight: 700,
+    lineHeight: "5rem" /* 125% */,
+    letterSpacing: "-1.408px",
+  },
+
+  h1: {
+    fontSize: ["2.75rem", null, null, "3rem"], // Breakpoint at lg
+    fontWeight: 700,
+    lineHeight: "3.5rem" /* 116.667% */,
+    letterSpacing: ["-0.0605rem", null, null, "-0.066rem"],
+  },
+
+  h2: {
+    fontSize: "2rem",
+    fontWeight: 700,
+    lineHeight: "2.5rem" /* 125% */,
+    letterSpacing: "-0.044rem",
+  },
+
+  h3: {
+    fontSize: "1.625rem",
+    fontWeight: 700,
+    lineHeight: "2rem" /* 123.077% */,
+  },
+
+  "subtitle-1": {
+    fontSize: "1.125rem",
+    fontWeight: 400,
+    lineHeight: "1.75rem" /* 155.556% */,
+    letterSpacing: "-0.02475rem",
+  },
+
+  "subtitle-2": {
+    fontSize: "1rem",
+    fontWeight: 400,
+    lineHeight: "1.375rem",
+    letterSpacing: "0.09375rem",
+    textTransform: "uppercase",
+  },
+
+  "body-1": {
+    fontSize: "1.25rem",
+    fontWeight: 400,
+    lineHeight: "2rem" /* 160% */,
+  },
+
+  "body-2": {
+    fontSize: "1.125rem",
+    fontWeight: 400,
+    lineHeight: "1.75rem" /* 160% */,
+  },
+
+  "link-button": {
+    textAlign: "center",
+    fontSize: "1.125rem",
+    fontWeight: 600,
+    lineHeight: "2rem" /* 177.778% */,
+    textTransform: "uppercase",
+    textDecoration: "underline",
+  },
+
+  "button-text": {
+    textAlign: "center",
+    fontSize: "1rem",
+    fontWeight: 600,
+    lineHeight: "2rem" /* 200% */,
+    textTransform: "uppercase",
+  },
+
+  link: {
+    fontSize: "1.125rem",
```

Comment:
Where did we get these styles from?

Reply:
Also copied over from our existing template, might need to change depending on the sgds revamp